recipe/meta.yaml

```diff
@@ -29,6 +28,7 @@ requirements:
     - cmake
     - libnetcdf 4.4.*
     - toolchain
+    - vc 14
```

Comment:
So my recommendation would be to make this `python` and add a `# [win]` selector to it.